app/views/casa_cases/_form.html.erb

```diff
@@ -12,6 +12,16 @@
         <% end %>
       </div>
 
+      <div class="field form-group">
+        <%= form.label :hearing_type_id %>
+        <%= form.collection_select(
+          :hearing_type_id,
+          HearingType.active.for_organization(current_user.casa_org),
```

Comment:
![stackerror](https://user-images.githubusercontent.com/21108297/96822194-7e322f00-13f7-11eb-9ca8-5fb4f3e13c0d.png)
Getting a `stack level too deep`error here if using `current_organization` for some reason. Tried to debug briefly with Jeanine in Slack with no luck as of yet. Open to suggestions.

Reply:
hmm weird. probably a factories problem. We should fix this
I made https://github.com/rubyforgood/casa/issues/1166 to track it, feel free to pick it up or not  :)